docs/modules/install/pages/empty-database.adoc

```diff
@@ -0,0 +1,32 @@
+= Install with no seeds
+
+You may want to setup your decidim with an empty database.
+In this case, please follow the steps bellow:
+
+. Follow steps 1 through 5 of previous section(xref:install:manual.adoc[Manual Installation] section).
+
+. Run the migrations from the root of your project in the terminal.
+[source,bash]
+----
+  bin/rails db:create db:migrate
```

Comment:
Let's remove the indentation, please! We should probably improve the general display of code in the docs (indenting a big, making the code highlight work, etc), but having these leading spaces in the beginning of the line has an unwanted sideffect with bash. By default in most systems that I've seen this is enabled by default (from `man bash`): 

> 
>        HISTCONTROL
>               A  colon-separated list of values controlling how commands are saved on the history list.  If the list of values includes ignorespace, lines
>               which begin with a space character are not saved in the history list.

You can try this with 

```bash
echo "hello world"
   echo "evil thing"
history | tail -n 2 
```

Assuming you're running bash, of course.

```suggestion
bin/rails db:create db:migrate
```